packages/gemacs.rb

```diff
@@ -3,47 +3,67 @@
 class Gemacs < Package
   description 'An extensible, customizable, free/libre text editor - and more.'
   homepage 'https://www.gnu.org/software/emacs/'
-  @_ver = '28.0.50.1'
-  version "#{@_ver}-01b0"
+  version '28.2'
   license 'GPL-3+, FDL-1.3+, BSD, HPND, MIT, W3C, unicode, PSF-2'
   compatibility 'all'
-  source_url 'https://git.savannah.gnu.org/git/emacs.git'
-  git_branch 'feature/pgtk'
-  git_hashtag '01b0a909b5ca858a09484821cc866127652f4153'
+  source_url 'https://ftpmirror.gnu.org/emacs/emacs-28.2.tar.xz'
+  source_sha256 'ee21182233ef3232dc97b486af2d86e14042dbb65bbc535df562c3a858232488'
 
   binary_url({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gemacs/28.0.50.1-01b0_armv7l/gemacs-28.0.50.1-01b0-chromeos-armv7l.tpxz',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gemacs/28.0.50.1-01b0_armv7l/gemacs-28.0.50.1-01b0-chromeos-armv7l.tpxz',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gemacs/28.0.50.1-01b0_i686/gemacs-28.0.50.1-01b0-chromeos-i686.tpxz',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gemacs/28.0.50.1-01b0_x86_64/gemacs-28.0.50.1-01b0-chromeos-x86_64.tpxz'
+       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gemacs/28.2_i686/gemacs-28.2-chromeos-i686.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/gemacs/28.2_x86_64/gemacs-28.2-chromeos-x86_64.tar.zst'
   })
   binary_sha256({
-    aarch64: '8ba0433113a23361cf961d372c8e7db79206c5baf3046255bebec0acd1d73452',
-     armv7l: '8ba0433113a23361cf961d372c8e7db79206c5baf3046255bebec0acd1d73452',
-       i686: 'e1a0efa63cf40683ddfd90ca181f43e50d87bbf6179774f976e6cc8f917f0205',
-     x86_64: '8f462f3de1a19f67a36e6752124da213add709a075cd8bc52815452d9bf68b16'
+       i686: 'd3c1c2133ca735512de8b77069093f78110a41377962080a69271d5052d18a46',
+     x86_64: 'ed1a8eacde7a8a17be9fb11a2bcdf83d3ce03ba3d78ebbb71884f808f515debd'
```

Comment:
Missing arm build?

Reply:
depends upon `webkit2gtk_4`, which isn't building for `armv7l`.

Reply:
adjusted compatibility for now